aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md

```diff
@@ -1,117 +1,143 @@
 msc.legacyurl: /signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role
 msc.type: authoredcontent
 ---
-Using SignalR Performance Counters in an Azure Web Role
-====================
-by [Tom FitzMacken](https://github.com/tfitzmac)
 
-> This article describes how to install and use SignalR performance counters in an Azure Web Role.
+# Using SignalR performance counters in an Azure Web Role
 
+By [Tom FitzMacken](https://github.com/tfitzmac)
 
-SignalR performance counters can be used to monitor performance in an Azure Web Role. The resulting counters will be captured by Windows Azure Diagnostics. Installing SignalR performance counters on Azure is done with SignalR.exe, the same tool that is used for standalone or on-premises applications. However, since Azure roles are transient, the application must be configured to install and register SignalR performance counters upon startup.
+SignalR performance counters are used to monitor your application's performance in an Azure Web Role. The counters are captured by Microsoft Azure Diagnostics. You install SignalR performance counters on Azure with *signalr.exe*, the same tool used for standalone or on-premises apps. Since Azure roles are transient, you configure an app to install and register SignalR performance counters upon startup.
 
-This tutorial demonstrates how to create an Azure Web Role application that uses SignalR 2 performance counters.
+## Prerequisites
 
-**Prerequisites**
-
-- [Visual Studio 2013](https://www.visualstudio.com/products/visual-studio-express-vs)
-- [Windows Azure SDK for Visual Studio 2013](https://www.windowsazure.com/en-us/downloads/)
-- A Microsoft Azure subscription. To sign up for a free Azure trial account, see [Azure Free Trial](https://azure.microsoft.com/en-us/pricing/free-trial/).
+* [Visual Studio 2015](https://www.visualstudio.com/vs/visual-studio-express/)
+* [Microsoft Azure SDK for Visual Studio 2015 (VS2015)](https://azure.microsoft.com/downloads/) **Note: Restart your dev machine after installing the SDK.**
+* Microsoft Azure subscription: To sign up for a free Azure trial account, see [Azure Free Trial](https://azure.microsoft.com/free/).
 
 ## Creating an Azure Web Role application that exposes SignalR performance counters
 
-1. Open Visual Studio 2013 with Administrator privileges.
-2. In Visual Studio 2013, select **File**, **New**, **Project**.
-3. In the **Templates** pane of the **New Project** window, under the **Visual C#** node, select the **Cloud** node, and select the **Windows Azure Cloud Service** template. Name the application **SignalRPerfCounters** and click **OK**.
+1. Open Visual Studio 2015.
+
+2. In Visual Studio 2015, select **File &gt; New &gt; Project...**.
+
+3. In the **Templates** pane of the **New Project** window under the **Visual C#** node, select the **Cloud** node and select the **Azure Cloud Service** template. Name the application **SignalRPerfCounters** and click **OK**.
```

Comment:
Buttons are *selected* and not *clicked*? Seems like if the item is something that highlights in the UI it's being *selected*. If it isn't highlighted but causes an action to occur, then it's *clicked*.